File: achieve/code_test/programmers/division.py
def solution(arr, divisor):
    answer = []
    for i in arr:
        if i % divisor == 0:
            answer.append(i)
    if len(answer) > 0:
        return sorted(answer)
    return [-1]


print(solution([5, 9, 7, 10], 5))

File: achieve/code_test/programmers/division.js
function solution(arr, divisor) {
  var answer = [];
  for (let i of arr) {
    if (i % divisor === 0) {
      answer.push(i);
    }
  }
  if (answer.length > 0) {
    return answer.sort((a, b) => a - b);
  }
  return [-1];
}

console.log(solution([5, 9, 7, 10], 5));
